Trippy/Utils/MapUtils.m

```diff
@@ -41,12 +41,11 @@ + (NSString *)generateOptimizedDirectionsApiUrl:(LocationCollection *)collection
     NSString *stops = @"optimize:true";
     int count = 0;
     for (Location *loc in collection.locations) {
-        if ([omitWaypoints containsObject:@(count)]) {
-            count += 1;
+        count += 1;
+        if ([omitWaypoints containsObject:@(count-1)]) {
```

Comment:
Oh I see, that is clumsier huh. 
...
Oh! I know! Try this (test it 😄):
```
if ([omitWaypoints containsObject:@(count++)]) {
```
That should increment the count after you read it for the index.